### ISSUE TITLE
Add an ability to provide additional environment variables for commanline

### ISSUE THREAD

**Issue by baev** [jenkinsci/allure-plugin#91]

**Comment by andrejska** [comment]
Will be this feature also include possibility to pass options flag, like `--config`?
As this feature has been implemented:
https://github.com/allure-framework/allure2/issues/378
Now we have possibility to have allure configuration file in project, but it cannot be used in scripted pipelines with allure jenkins plugin :(

**Comment by eroshenkoam** [comment]
@andrejska no, it's about variables, I will make another ticket for your request

**Comment by mikesalvia** [comment]
`--config` option was added in PR #204 it is still under review.

**Comment by mikesalvia** [comment]
@andrejska this should address your request but doesn't allow for env vars

**Comment by baev** [comment]
Environment-variable support implemented in 2.32.0 via [#371](https://github.com/jenkinsci/allure-plugin/pull/371)
